test/selenium/lib/view/signed_out/accessors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+var By = require('selenium-webdriver').By;
+
+var SELECTORS = Object.freeze({
+    page: By.id('signin')
+});
+
+function SignedOutAccessor(driver) {
+  this.driver = driver;
+};
+
+SignedOutAccessor.prototype = {
+   get isSignOutPage(){
+   	  return this.driver.isElementPresent(SELECTORS.page);
```

Comment:
A little bit better: get root() {
   	var signinPage = this.driver.findElement(SELECTORS.page);
  	return this.driver.wait(webdriver.until.elementIsVisible(signinPage));
   }
Should this be valid?